src/main/features/response/routes/when-will-you-pay.ts

```diff
@@ -0,0 +1,53 @@
+import * as express from 'express'
+
+import { Paths } from 'response/paths'
+
+import { ErrorHandling } from 'common/errorHandling'
+import { DefendantPaymentOption, DefendantPaymentType } from 'response/form/models/defendantPaymentOption'
+import { Form } from 'forms/form'
+import { FormValidator } from 'forms/validation/formValidator'
+import { User } from 'idam/user'
+import { DraftService } from 'services/draftService'
+
+function renderView (form: Form<DefendantPaymentOption>, res: express.Response) {
+  const user: User = res.locals.user
+  res.render(Paths.defenceFullPartialPaymentOptionsPage.associatedView, {
+    form: form,
+    claim: user.claim
+  })
+}
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.defenceFullPartialPaymentOptionsPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
+      const user: User = res.locals.user
+      const defendantPaymentOption: DefendantPaymentOption = user.responseDraft.document.defendantPaymentOption
+
+      renderView(new Form(defendantPaymentOption), res)
+    }))
+  .post(Paths.defenceFullPartialPaymentOptionsPage.uri,
+    FormValidator.requestHandler(DefendantPaymentOption, DefendantPaymentOption.fromObject),
+    ErrorHandling.apply(
+      async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
+        const form: Form<DefendantPaymentOption> = req.body
+        const user: User = res.locals.user
+
+        if (form.hasErrors()) {
+          renderView(form, res)
+        } else {
+          user.responseDraft.document.defendantPaymentOption = form.model
+          await new DraftService().save(user.responseDraft, user.bearerToken)
+
+          const { externalId } = req.params
+
+          switch (form.model.option) {
+            case DefendantPaymentType.FULL_BY_SPECIFIED_DATE:
+              res.redirect(Paths.defenceFullPartialPaymentOptionsPage.evaluateUri({ externalId: externalId }))
```

Comment:
Take user to not implemented page instead of redirecting to the same page. This way it will be easier to understand this is not implemented yet and has to be worked on laster.

Reply:
ok

Reply:
Can't see a not implemented page, we used to use 'ToDO' before